helm_deploy/values-dev.yaml

```diff
@@ -12,6 +12,8 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     API_BASE_URL_OAUTH: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     API_BASE_URL_NOMIS: https://nomis-prsner-dev.aks-dev-1.studio-hosting.service.justice.gov.uk
+    # Set to ourselves until new service is ready
```

Comment:
since real API is only partially developed we will point back to this service and create temporary stub APIs